fluent-plugin-google-cloud.gemspec

```diff
@@ -10,7 +10,7 @@ eos
   gem.homepage      =
     'https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud'
   gem.license       = 'Apache-2.0'
-  gem.version       = '0.12.9'
+  gem.version       = '0.12.10'
```

Comment:
The version bump should have been handled by release automation already, right?

https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/releases (See the last released version is 0.12.8)